eb

```diff
@@ -60,15 +60,6 @@ then
     exit 2
 fi
 
-# support for Python versions older than v2.6 is deprecated
```

Comment:
there's no deprecated Python version anymore now, using something older than Python 2.6 will result in an error message, see the block above

so, this has become useless